src/main/cpp/t04_both2.cpp

```diff
@@ -27,5 +27,40 @@
 using namespace std;
 
 int t04_both2() {
+	set <int> set1, set2, set3;
+    int n, m, cou = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'cou' is assigned a value that is never used.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4695222)